feature/bgp/policybase/otg_tests/aspath_and_community_test/README.md

```diff
@@ -0,0 +1,109 @@
+# RT-2.4: BGP Policy AS Path Set and Community Set
+
+## Summary
+
+BGP policy configuration for AS Paths and Community Sets
+
+## Procedure
+
+* RT-2.4.1 - Test setup
+  * Use cfgLib to generate config for 2 DUT ports
+  * Use cfgLib to generate config for DUT port 1 eBGP session
+
+  * Use helper to configure ATE 2 ports
+  * Use helper to configure ATE port 1 with 1 eBGP session
+
+  * Establish eBGP sessions between ATE port-1 and DUT port-1
+  * For IPv4 and IPv6 routes:
+    * Advertise IPv4 prefixes over IPv4 neighbor from ATE port-1, observe
+            received prefixes at ATE port-2.
+    * Advertise IPv6 prefixes over IPv6 neighbor from ATE port-1,
+        observe received prefixes at ATE port-2.
+  * Generate traffic from ATE port-2 to ATE port-1
+  * Validate that traffic can be received on ATE port-1 for all installed
+        routes
+
+* RT-2.4.1 - Validate single routing-policy containing as-path-set and ext-community-set
+  * Configure ATE
+    * Use paths and routes from RT-2.2.3 and RT-2.2.4 where each route have both aspath and ext-community set.
+  * Configure DUT
+    * Replace the DUT's bgp import-policy configuration for the bgp neighbor ATE port1 to use as-path-set-name `my_regex_aspaths` and community-set-name `my_regex_comms`
```

Comment:
`my_regex_aspaths` and community-set-name `my_regex_comms`do not exist

Please add verification base on RIB view (similiarly to RT-2.3 and RT-2.2). Please specify explicitly what is expected.